CHANGELOG.md

```diff
@@ -5,6 +5,143 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v8.0.0-alpha.1
+
+_Nov 22, 2024_
+
+We'd like to offer a big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔧 Refactor Tooltip customisation for charts -  — [Learn more](https://next.mui.com/x/react-charts/tooltip/#overriding-content)
+- 🌍 Improve Chinese, Spanish, and Swedish locale on the Data Grid component
```

Comment:
```suggestion
- ⚛️ React 19 support
- 🌍 Improve Chinese, Spanish, and Swedish locale on the Data Grid component
```

Reply:
Not sure if [the PR](https://github.com/mui/mui-x/pull/15342) should be linked here or in the each component

Reply:
I would be fine duplicating it for each set of components.
Usually we would put it in `[core]` if it was cross component, but that's too impactful to be in `[core]` imho.

And of course with an highlight :+1:

Reply:
Added suggestion for a new entry for each component